nemo/collections/nlp/modules/common/sequence_token_classifier.py

```diff
@@ -51,7 +51,7 @@ def __init__(
         """
         Initializes the SequenceClassifier module.
         Args:
-            hidden_size: the hidden size of the mlp head on the top of the encoder
+            hidden_size: hidden size of the mlp head on the top of the encoder
```

Comment:
@VahidooX I thought we wanted to break down intent-slot to have 2 separate heads? But even master has everything combined. I'm fine with leaving it as for this PR, but it should be fixed.
@vladgets this file should be moved out of the common, it's intent-slot specific

Reply:
How is it different from "sequence_classifier.py" and "token_classifier.py" with both of them sitting in common folder too?

Reply:
Both sequence_classifier and token_classifier are general modules and can be used in multiple tasks (please take a look at BertPretraining and Punctuation models where those modules are used). Ideally, your intent-slot model should utilize those modules and not introduce a new module  that is just a combination of the existing modules. The module you're adding will only work for intent-slot task, that's why it shouldn't be part of common.